MachObfuscator/SymbolMangling/RealWordsMangler/SentenceGenerator.swift

```diff
@@ -6,7 +6,11 @@ import Foundation
 // with
 // 1-* nouns https://www.talkenglish.com/vocabulary/top-1500-nouns.aspx
 
-class SentenceGenerator {
+protocol SentenceGenerator {
```

Comment:
Hmm, I thought about protocols in the opposite way. I always leave protocol declared as generic as it can be, every time I was thinking: "maybe someone will conform a struct to the protocol someday.". Does restraining the protocol to reference types adds any value?